### ISSUE TITLE
feat: privacy and security

### ISSUE THREAD

**Issue by cablehead** [cablehead/stacks#66]
Stacks should give users as much control as possible to prevent data they don't want being stored out of the clip store, and make it as easy as possible to location, and permanently delete data, for sensitive data that's accidentally captured.

For reference these are the options the excellent PastePal gives you h/t @0n0mat0p03ia

<img width="300" alt="image" src="https://github.com/cablehead/stacks/assets/1394/9aa0e4cc-4894-4442-aa1d-76b4a1a2f32b">

I think these are our available tools to prevent sensitive data entering the clipboard:

- respect http://nspasteboard.org
- allow users to specify apps that shouldn't be captured from
- allow users to specify patterns to ignore. look to Atuin for inspiration for default filters https://docs.atuin.sh/configuration/config/#secrets_filter

I think the clip store api might be simplest way to let users locate data across stacks (and versions of clips that may now be obscured, after being edited).  https://stacks.cross.stream/releases/v0.15.6

**Comment by cablehead** [comment]
it looks like the mac keychain doesn't set org.nspasteboard.ConcealedType 😭 Although my password manager does. The better bang for buck would be to add app filtering first

**Comment by slidingshade** [comment]
Yeah, app filtering to ignore password managers sounds reasonable.  Also everything from  Universal Clipboard (shared from iphone/ipad devices) automatically goes to stacks, great to have option to disable this (dunno if its possible).

**Comment by cablehead** [comment]
If we can pick up a mac specialist to look at [ability to paste directly from Stacks](https://github.com/cablehead/stacks/issues/47) and [improve window "spotlight"-ness](https://github.com/cablehead/stacks/issues/72), they might be able to advice us on adding an option to ignore the Universal Clipboard (with a corresponding phone app yet, I currently really like being able to get clips via Universal Clipboard 😅)
